pkg/platform/github.go

```diff
@@ -17,62 +17,67 @@ package platform
 import (
 	"context"
 	"fmt"
-	"time"
 
-	"github.com/google/go-github/v53/github"
-	"golang.org/x/oauth2"
-
-	gh "github.com/abcxyz/guardian/pkg/github"
-	"github.com/abcxyz/pkg/githubauth"
+	"github.com/abcxyz/guardian/pkg/github"
+	"github.com/abcxyz/pkg/logging"
 )
 
 var _ Platform = (*GitHub)(nil)
 
 // GitHub implements the Platform interface.
 type GitHub struct {
-	cfg    *gh.Config
-	client *github.Client
+	cfg    *github.Config
+	client github.GitHub
```

Comment:
@verbanicm I've changed my approach a bit here. I think we need to keep the `github` package for mocking the GitHub client.